### DISCUSSION TITLE
Weird Sysex Messages coming from Opendeck software flashed BlackPill

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#304] [Q&A]
Hello all!

I have finished (or am finishing) my first build of a midi controller based on the BlackPill and Opendeck software.

It's pretty rudimentary. Four potentiometers and a momentary button, plugged into my MacBook via the Blackpill USB connector.

The pots are two slide pots, one rotary pot, and one TRS jack to run an expression pedal.

The button is on a TS jack to plug in a momentary foot-pedal.

All is working fine, but I am getting the occasional random sysex message from the unit.

It seems to be the same message each time, and occurs when I am moving any of the pots (the slide and rotary). I have not tried the TRS and TS jacks yet.

The message is F0 00 53 43 01 00 49 03 00 02 F7.

I'm not concerned with what that syex message represents, just why am I getting any sysex message at all and how can I stop the controller from doing that?

I opened a Midi Monitor application and have attached the images below.

Thanks!

![Opendeck Sysex Issue](https://github.com/user-attachments/assets/0137c855-2d3e-4499-8b37-069f9381268b)
![Opendeck Sysex Issue 4](https://github.com/user-attachments/assets/1d6f22ec-a9eb-4372-bf77-f5a57901004d)
![Opendeck Sysex Issue 3](https://github.com/user-attachments/assets/d667e520-c79a-47b5-bdf0-718c0691da62)
![Opendeck Sysex Issue 2](https://github.com/user-attachments/assets/f7579e6a-9ed2-47ff-9586-16c76df5a462)

**Comment by paradajz** [comment]
Those messages are sent once the web configurator opens the connection with OpenDeck. Documented here:
https://github.com/shanteacontrols/OpenDeck/wiki/SysEx-Configuration#component-info-messages

**Comment by NRKorsakov** [comment]
Hi thanks so much!

Working like a charm now.

And again, thank you for explaining things to those of us who are non-technically-minded.

I (and I'm sure there are many others) really appreciate it and all of what you are so kindly doing for all of us here!
